doc/config/vhosts.d/vhosts.template

```diff
@@ -21,14 +21,13 @@ $HTTP["host"] == "download.example.com" {
   accesslog.filename          = log_root + "/" + server_name + "/access.log"
   server.errorlog             = log_root + "/" + server_name + "/error.log"
 
-  $HTTP["scheme"] == "https" {
-    server.name = "download.example.com"
-    ssl.pemfile = "/etc/lighttpd/certs/download.example.com.cert.pem"
-    ssl.privkey = "/etc/lighttpd/certs/download.example.com.key.pem"
-    ssl.engine  = "enable"
+#  $HTTP["scheme"] == "https" {
+#    server.name = "download.example.com"
+#    ssl.pemfile = "/etc/lighttpd/certs/download.example.com.cert.pem"
+#    ssl.privkey = "/etc/lighttpd/certs/download.example.com.key.pem"
 
-    server.document-root = vhosts_dir + "/example.com/download/htdocs"
```

Comment:
Why should this section be here at all?  I already questioned that `$HTTP["scheme"]` is redundant, and you have not addressed that.  What value is changing the server.document-root for https, and why does it belong in vhosts.template?  Keep in mind that the historical doc was written long ago, and I am questioning whether or not it is still relevant today, when most sites are using TLS.

Reply:
Should I remove this completely or just the `server.document-root` line? I kept this as an example. I should've changed the value.

Reply:
Since you have not provided a reasoning why you desire changes here, you should remove all of your changes here, in this part of the patch.  Since I have presented a reason why I plan to make changes, I will (separately) make the change for which I have shared my reasoning.  Put another way, when you are making contributions, please carefully scope your changes to changes that you are comfortable explaining and justifying.

Reply:
Sure, thanks.